From 01da6074e5379baa01b5433e46a35b98110f641a Mon Sep 17 00:00:00 2001
From: Cheick TRAORE <cheick.traore-ext@st.com>
Date: Thu, 28 Mar 2024 13:43:45 +0100
Subject: [PATCH 1004/1141] ARM: configs: multi_v7_addons: Enable IRQ in
 debugfs for stm32mp1

Add and enable CONFIG_GENERIC_IRQ_DEBUGFS in multi_v7 addons
fragment file.

Change-Id: I184c329fd4cc6bcdcd09c72201c31ae7d21026f2
Signed-off-by: Cheick TRAORE <cheick.traore-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/369937
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -33,6 +33,7 @@ CONFIG_FORCE_MAX_ZONEORDER=12
 CONFIG_FUTEX=y
 CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 CONFIG_GDB_SCRIPTS=y
+CONFIG_GENERIC_IRQ_DEBUGFS=y
 CONFIG_HIGHMEM=y
 CONFIG_HWSPINLOCK=y
 CONFIG_HWSPINLOCK_STM32=y
